vocode/streaming/streaming_conversation.py

```diff
@@ -589,6 +617,39 @@ def is_interrupt(self, transcription: Transcription):
         return transcription.confidence >= (
             self.transcriber.get_transcriber_config().min_interrupt_confidence or 0
         )
+    
+
+
+    async def send_noise(self):
+        filler_audio_noise = FillerAudio(
+                        BaseMessage(text="Noise"),
+                        audio_data=convert_wav(
+                            '/Users/mohammadaminroohi/miniconda3/envs/vocode_1/lib/python3.9/site-packages/vocode/streaming/pink BL.wav',
```

Comment:
This is referencing a file that is not present in the source. Consider adding it to this PR.